CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 1.15
 
+## 1.15.2 - 2024-06-20
+
+### Fixed
+
+- Remove unexpected `altname` after renaming interface to avoid failure when moving host interface to OVS bridge. ([#6321](https://github.com/antrea-io/antrea/pull/6321), [@gran-vmv])
+- Do not try to update type of Secret in selfSignedCertProvider to avoid self-signed certificate update failure. ([#6205](https://github.com/antrea-io/antrea/pull/6205), [@tnqn])
+- Install multicast related iptables rules only on IPv4 chains to fix the antrea-agent initialization failure occurred when the Multicast feature is enabled in dual-stack clusters. ([#6123](https://github.com/antrea-io/antrea/pull/6123), [@wenyingd])
```

Comment:
s/occurred/occurring

but I see that we have the same typo for Antrea v2.0 so we may actually want to keep it the same

Reply:
Thanks, fixed typo, I feel no need to keep the same for a typo : )